tools/BUILD

```diff
@@ -23,12 +23,6 @@ py_binary(
     srcs = ["wheelmaker.py"],
 )
 
-# These files are generated and updated by ./update_tools.sh
-exports_files([
-    "piptool.par",
-    "whltool.par",
```

Comment:
I bet there's also some docs or checks for these files we can remove

Reply:
You were right. I found the CONTRIBUTING docs and removed them.